db/migrate/20150502105558_create_events.rb

```diff
@@ -0,0 +1,12 @@
+class CreateEvents < ActiveRecord::Migration
+  def change
+    create_table :events do |t|
+      t.string :title
+      t.text :description
+      t.datetime :start
```

Comment:
if you have an attribute as date time, we're naming the attribute with ending '_at', so here the attribute name should be 'start_at'. If we had the attribute as date, then we would name it as 'start_date'. please make and send the changes